tests/io/lobster/test_inputs.py

```diff
@@ -2590,7 +2588,7 @@ def test_raises(self):
             self.hamilton_matrices = LobsterMatrices(filename=f"{TEST_FILES_DIR}/cohp/Na_hamiltonMatrices.lobster.gz")
 
         with pytest.raises(
-            OSError,
+            RuntimeError,
```

Comment:
i think `IOError` would be the most fitting built-in exception class here

Reply:
Hi Janosh. Thanks for reviewing! However I don't quite agree on the usage of `IOError` in this case, similar to my reason against `OSError` here https://github.com/materialsproject/pymatgen/pull/3757#issuecomment-2054362292.

Maybe I don't understand it correctly, but I think `IOError` should be used in cases where there is a IO issue. For some reason the official docs doesn't give details on this, but from [this](https://www.tutorialspoint.com/How-to-catch-IOError-Exception-in-Python):
> IOError Exception

> It is an error raised when an input/output operation fails, such as the print statement or the open() function when trying to open a file that does not exist. It is also raised for operating system-related errors.

In our case, the file opens just fine but the content is empty, so I don't think it's actually an IO issue.

Reply:
fair enough, i'm definitely ok with `RuntimeError`

Reply:
All good!